tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -235,82 +323,7 @@ describe('Target aggregates', () => {
     it('should display correct data', async () => {
```

Comment:
Can you confirm that the side filter doesn't show if a user has only one facility? Does it make sense to add a validation in this scenario or create a new one to test this specifically? Let me know what you think.
cc: @latin-panda

Reply:
Yes, the sidebar should not appear if the user has only 1 facility assigned. So, it's better to have a scenario for the user with 1 facility and another scenario for the user with 2 or 3 facilities.